libs/oidc/provider/src/lib/entities/oidc-client.entity.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  TableForeignKey,
+  ManyToOne,
+  OneToMany,
+  JoinColumn,
+} from "typeorm";
+import { IRequiredEntityAttrs } from "./entity.interface";
+import { GuidIdentity } from "./guid-identity.entity";
+
+@Entity({
+  name: "client_metadata",
+})
+export class ClientMetadata extends GuidIdentity {
```

Comment:
Imports

Members access

Some of these missing Column decorators?

Run prettier.